cockroachdb/goose/migration.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"context"
+	"database/sql"
+	"runtime"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/stdlib"
+	"github.com/pressly/goose/v3"
+)
+
+type PgxContextMigration func(context.Context, *pgx.Conn) error
+
+func AddPgxContextMigration(upMigration PgxContextMigration, downMigration PgxContextMigration) {
+	// runtime.Caller returns the filename of the migration file that calls this function
+	_, filename, _, _ := runtime.Caller(1)
```

Comment:
I have a linting issue with dogsled on this line
It is described here: https://github.com/alexkohler/dogsled/issues/3